blast/2.13.0/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:focal as app
+
+ARG BLAST_VER="2.13.0"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+LABEL base.image="ubuntu:focal"
+LABEL dockerfile.version="1"
+LABEL software="blast+"
+LABEL software.version=$BLAST_VER
+LABEL description="Finds matches in sequencing reads"
+LABEL website="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastDocs&DOC_TYPE=Download"
+LABEL license="https://www.ncbi.nlm.nih.gov/IEB/ToolBox/CPP_DOC/lxr/source/scripts/projects/blast/LICENSE"
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# RUN executes code during the build
+RUN apt-get update && apt-get install -y \
+  libgomp1 \
```

Comment:
just curious, what is `libgomp1` included for?

Reply:
According to the documentation:
https://www.ncbi.nlm.nih.gov/books/NBK569857/

And my own observations:
```
Step 16/19 : FROM app as test
 ---> 44e564dcf950
Step 17/19 : RUN mkdir db &&   wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/005/845/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz -P db &&   gunzip db/GCF_000005845.2_ASM584v2_genomic.fna.gz &&   makeblastdb -dbtype nucl -in db/GCF_000005845.2_ASM584v2_genomic.fna
 ---> Running in 02612acccc4e
--2022-05-17 22:19:57--  https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/005/845/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz
Resolving ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)... 130.14.250.10, 130.14.250.11, 2607:f220:41e:250::7, ...
Connecting to ftp.ncbi.nlm.nih.gov (ftp.ncbi.nlm.nih.gov)|130.14.250.10|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1379902 (1.3M) [application/x-gzip]
Saving to: 'db/GCF_000005845.2_ASM584v2_genomic.fna.gz'

     0K .......... .......... .......... .......... ..........  3%  495K 3s
    50K .......... .......... .......... .......... ..........  7% 1.06M 2s
   100K .......... .......... .......... .......... .......... 11% 17.2M 1s
   150K .......... .......... .......... .......... .......... 14%  984K 1s
   200K .......... .......... .......... .......... .......... 18% 88.9M 1s
   250K .......... .......... .......... .......... .......... 22%  216M 1s
   300K .......... .......... .......... .......... .......... 25%  206M 1s
   350K .......... .......... .......... .......... .......... 29% 1.11M 1s
   400K .......... .......... .......... .......... .......... 33% 12.1M 0s
   450K .......... .......... .......... .......... .......... 37% 99.1M 0s
   500K .......... .......... .......... .......... .......... 40%  128M 0s
   550K .......... .......... .......... .......... .......... 44%  128M 0s
   600K .......... .......... .......... .......... .......... 48%  333K 0s
   650K .......... .......... .......... .......... .......... 51%  255M 0s
   700K .......... .......... .......... .......... .......... 55%  251M 0s
   750K .......... .......... .......... .......... .......... 59%  244M 0s
   800K .......... .......... .......... .......... .......... 63%  212M 0s
   850K .......... .......... .......... .......... .......... 66%  251M 0s
   900K .......... .......... .......... .......... .......... 70%  251M 0s
   950K .......... .......... .......... .......... .......... 74%  242M 0s
  1000K .......... .......... .......... .......... .......... 77%  223M 0s
  1050K .......... .......... .......... .......... .......... 81%  267M 0s
  1100K .......... .......... .......... .......... .......... 85%  275M 0s
  1150K .......... .......... .......... .......... .......... 89%  252M 0s
  1200K .......... .......... .......... .......... .......... 92%  199M 0s
  1250K .......... .......... .......... .......... .......... 96%  255M 0s
  1300K .......... .......... .......... .......... .......   100%  253M=0.4s

2022-05-17 22:19:58 (3.26 MB/s) - 'db/GCF_000005845.2_ASM584v2_genomic.fna.gz' saved [1379902/1379902]

makeblastdb: error while loading shared libraries: libgomp.so.1: cannot open shared object file: No such file or directory
The command '/bin/sh -c mkdir db &&   wget https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/005/845/GCF_000005845.2_ASM584v2/GCF_000005845.2_ASM584v2_genomic.fna.gz -P db &&   gunzip db/GCF_000005845.2_ASM584v2_genomic.fna.gz &&   makeblastdb -dbtype nucl -in db/GCF_000005845.2_ASM584v2_genomic.fna' returned a non-zero code: 127
```

Reply:
got it, thanks!